src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/model/incorporation/IncorporationKind.java

```diff
@@ -6,6 +6,7 @@
 public enum IncorporationKind {
     REGISTRATION("limited-partnership-registration"),
     TRANSITION("limited-partnership-transition"),
+    POST_TRANSITION("limited-partnership-post-transition"),
```

Comment:
Ah, I see these 'kind' strings aren't defined in the `Constants` class. Maybe they should be also or we clean this up later as existing code?

Reply:
Resolving. Maybe clean this up later.